### ISSUE TITLE
Avoid showing same WordPress post in multiple loops (easypost and repeater)

### ISSUE THREAD

**Issue by iscoseo** [soflyy/oxygen-bugs-and-features#575]
When we show a featured section with some posts. They are also shown later in other blocks. 

It would be great to have a check option that allows you to exclude posts that have been shown previously. 

https://www.isitwp.com/avoid-duplicate-posts-in-multiple-loops/

**Comment by Hanleymade** [comment]
This is a great feature to have on Newspaper or Blog templates. The Newspaper theme uses this feature well.

**Comment by iscoseo** [comment]
exactly, is a necessary function for the development of magazine style sites
